### DISCUSSION TITLE
How to enable?

### DISCUSSION THREAD

**Discussion by notjelly** [GriefPrevention/GriefPrevention#1119] [Q&A] [answered]
How to enable limit of hoppers and pistons in claim in config?

**Comment by Jikoo** [comment]
The active blocks inspection is part of creative claims mode. It isn't configurable by any other settings.

**Comment by notjelly** [comment]
Do I need an add-on that you can make? and I pay you?

**Comment by Jikoo** [comment, marked as answer]
You can enable creative claims mode per-world already [in the config](https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#limiting-which-worlds-griefprevention-runs-on).

If creative mode isn't exactly what you want and you're looking to pay someone to develop an addon, you should probably use [Robo's thread in Spigot's S&R](https://www.spigotmc.org/threads/requests-for-robomwm.293977/).
I personally do not do contract work for Bukkit plugin development. I accept unprompted monetary contributions after-the-fact, but I will not engage in programming for pay - this is a hobby for me, not a job.

Possibly related: #652 

/e whoops, didn't mean to close.

**Comment by Jikoo** [comment]
Going to assume that this question is answered due to lack of response. OP, if you do want additional options for active blocks, please post any requirements in #652.
